src/dlgTriggerEditor.cpp

```diff
@@ -345,7 +345,7 @@ dlgTriggerEditor::dlgTriggerEditor( Host * pH )
     connect( showSearchAreaAction, SIGNAL(triggered()), this, SLOT( slot_show_search_area()));
 
     QAction * saveAction = new QAction( QIcon( QStringLiteral( ":/icons/document-save-as.png" ) ), tr("Save Item"), this);
-    //saveAction->setShortcut(tr("Ctrl+S"));
+    saveAction->setShortcut(tr("Ctrl+S"));
```

Comment:
I suspect it might be a matter of the applicability of the shortcut i.e. the `QShortcut::context` property - according to the Qt Documentation:
> By default, this property is set to Qt::WindowShortcut.

and it says:

>For a `QEvent::Shortcut` event to occur, the shortcut's key sequence must be entered by the user in a context where the shortcut is active. The possible contexts are these:
* Qt::WidgetShortcut = 0: The shortcut is active when its parent widget has focus.
* Qt::WidgetWithChildrenShortcut = 3: The shortcut is active when its parent widget, or any of its children has focus. Children which are top-level widgets, except pop-ups, are not affected by this shortcut context.
* Qt::WindowShortcut = 1: The shortcut is active when its parent widget is a logical subwidget of the active top-level window.
* Qt::ApplicationShortcut = 2: The shortcut is active when one of the applications windows are active.

I wonder whether we need to use the non-default `Qt::WidgetShortcut`...